src/v/model/model.cc

```diff
@@ -189,10 +189,11 @@ std::istream& operator>>(std::istream& i, compression& c) {
 std::ostream& operator<<(std::ostream& o, const model::broker_properties& b) {
     fmt::print(
       o,
-      "{{cores {}, mem_available {}, disk_available {}}}",
+      "{{cores {}, mem_available {}, disk_available {}, in_fips_mode {}}}",
       b.cores,
       b.available_memory_bytes,
       b.available_disk_gb,
+      b.in_fips_mode,
       b.mount_paths,
       b.etc_props);
```

Comment:
weird...

Reply:
sorry, not related to this diff. these look like extra args that are not reflected in the format string 🤷